.travis.yml

```diff
@@ -20,6 +21,9 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew reinstall --with-pcre2 git; fi
 
+  # install npm tools for eslint
+  - cd ./frontend; npm install --only=dev; cd ..
```

Comment:
npm install --only=dev /frontend?

Reply:
not sure if that works, let me try.

Reply:
```
$ npm install --only=dev /frontend
npm ERR! code ENOLOCAL
npm ERR! Could not install from "../../../../frontend" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ongspxm/.npm/_logs/2018-06-29T03_15_16_215Z-debug.log
```

Reply:
`"../../../../frontend"` seems like its not using correct directory, try ./frontend